### ISSUE TITLE
Please just git rid of this repo 

### ISSUE THREAD

**Issue by SerpentDrago** [I-A-C/ExodusReduxRepo#24]
people are getting confused and posting issues into this repo . 

also i've found support issues where people would grab stuff from here or old kodi repo never updated . 

its a mess . I'd rather deal with posts about "the repo doesn't work" then it half working and grabbing old stuff . 
I say just nuke it !

**Comment by drinfernoo** [comment]
Yeah, it's probably time, but any applicable issues from this repo need to get migrated to their respective repository first.

**Comment by I-A-C** [comment]
I left a zip of the ExodusReduxRepo-master in the repository as a placeholder for now.  Wanted to make sure all the issues where addressed before completely deleting the whole thing.

**Comment by jewbmx** [comment]
Could do like my repo and just copy the repo code of both into both to link em, then make the old repo require the new one to be installed as well. After a week or so delete the old repo and whoever misses out on that update o well.

Its the one time i find the required feature pretty nice lol
